### PR TITLE
Allow PHP 8

### PR DESCRIPTION
**Pull request by ddeboer**
* Extend test matrix.
* Update dependencies to PHP 8-compatible versions.

### CODE REVIEW

**Comment by mnapoli** [comment]
It seems the build is not running on Travis CI :/

We may need to switch to GitHub Actions

**Comment by mnapoli** [comment]
Now that #41 is merged could you please add the CI job for running on PHP 8? Then I think we're good to merge.

**Comment by ddeboer** [comment]
Done!

**Comment by mnapoli** [comment]
Thanks, the build is failing however.

File: composer.json [hunk 1]
```diff
@@ -25,14 +25,15 @@
         "bref/bref": "^0.5.18 || ^1.0",
         "async-aws/sns": "^1.0",
         "async-aws/sqs": "^1.2",
-        "async-aws/event-bridge": "^1.0"
+        "async-aws/event-bridge": "^1.0",
+        "symfony/http-client": "<5.2"
```
Comment (jderusse) [inline composer.json:29]: `async-aws` release a versin that fix it https://github.com/async-aws/core/releases/tag/1.7.0
Could you have a try?

Comment (ddeboer) [reply composer.json]: Thanks, that was quick! 😄 Done.

**Comment by ddeboer** [comment]
All green now! 🌲

File: composer.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         }
     },
     "require": {
-        "php": "^7.3",
+        "php": "^7.3 || ^8.0",
```
Comment (shouze) [inline composer.json:18]: Please prefer this one:
```suggestion
        "php": ">=7.3",
```

Comment (shouze) [reply composer.json:18]: See [reasons in this tweet](https://twitter.com/nicolasgrekas/status/1263023258938548225).

**Review by shouze — APPROVED** [review]

File: composer.json [hunk 3]
```diff
@@ -28,11 +28,11 @@
...
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5",
-        "mnapoli/hard-mode": "^0.2.0",
+        "phpunit/phpunit": "^9.4",
+        "mnapoli/hard-mode": "^0.3.0",
         "phpstan/phpstan": "^0.12.0",
         "symfony/framework-bundle": "^4.3 || ^5.0",
-        "nyholm/symfony-bundle-test": "^1.6",
+        "nyholm/symfony-bundle-test": "dev-master",
```
Comment (Nyholm) [inline composer.json:35]: There is a tagged version now. 

```suggestion
        "nyholm/symfony-bundle-test": "^1.6",
```

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**